Dockerfile

```diff
@@ -9,5 +9,5 @@ ADD scm-source.json /scm-source.json
 EXPOSE 8080
 
 # run the server when a container based on this image is being run
-ENTRYPOINT java -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
+ENTRYPOINT java $JAVA_OPTS -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
```

Comment:
@rcillo not that I'm aware of. How is it possible to do that?

@antban Yes, it's one of these options, but I don't want to make it configurable I think it should always be there. I can move it to the left from -jar if you prefer :)